doc/source/user_guide/timeseries.rst

```diff
@@ -689,7 +691,11 @@ If the timestamp string is treated as a slice, it can be used to index ``DataFra
 
    To *always* have unambiguous selection, whether the row is treated as a slice or a single selection, use ``.loc``.
 
+   As of version 1.1.0, row-based indexing is deprecated.  Users should always use ``.loc``
+   for label-based row indexing.
+
    .. ipython:: python
+      :okwarning:
```

Comment:
Does this cause a warning?

Reply:
i took the kitchen-sink approach to getting the docbuild passing, will figure out which of these can be trimmed away